### PR TITLE
upgrade swagger-ui to 2.0.24

### PR DESCRIPTION
**Pull request by stuartf**

### CODE REVIEW

File: docs/rest/css/screen.css [hunk 1]
```diff
@@ -1125,7 +1128,7 @@
   height: 18px;
   vertical-align: middle;
   display: inline-block;
-  background: url(/docs/rest/images/explorer_icons.png) no-repeat;
+  background: url(../images/explorer_icons.png) no-repeat;
```
Comment (nicolaasmatthijs) [inline docs/rest/css/screen.css]: I think the absolute path is required here to ensure that this works on a production build. Can you verify that's the case?

File: docs/rest/css/screen.css [hunk 2]
```diff
@@ -1165,7 +1168,7 @@
   font-size: 1.5em;
   font-weight: bold;
   text-decoration: none;
-  background: transparent url(/docs/rest/images/logo_small.png) no-repeat left center;
+  background: transparent url(../images/logo_small.png) no-repeat left center;
```
Comment (nicolaasmatthijs) [inline docs/rest/css/screen.css]: Same here

File: docs/rest/css/screen.css [hunk 3]
```diff
@@ -743,17 +743,20 @@
...
-  float: right;
-}
 .swagger-section .swagger-ui-wrap ul#resources li.resource ul.endpoints li.endpoint ul.operations li.operation div.content div.sandbox_header input.submit {
   display: block;
   clear: none;
   float: left;
   padding: 6px 8px;
 }
+.swagger-section .swagger-ui-wrap ul#resources li.resource ul.endpoints li.endpoint ul.operations li.operation div.content div.sandbox_header span.response_throbber {
+  background-image: url('../images/throbber.gif');
```
Comment (nicolaasmatthijs) [inline docs/rest/css/screen.css]: And here

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @stuartf for follow-up.

@stuartf : Are there any back-end changes associated to this PR?

**Comment by stuartf** [comment]
This should work with the current backend and make us ready for upgrading the backend doc to swagger 2.0.

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged

**Pull request merged**